### PR TITLE
feat(FDS-519): [Atoms] Add datepicker component

### PR DESCRIPTION
**Pull request by rociobaena**
Create a date picker component
This component will be exported in @espressive/cascara
Fixture and documentation added.

image

### Dependencies
Antd has been added

## PR Author Checklist

- [x] PR title adheres to [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#specification)
- [x] I have finished reviewing the contents of this PR for accuracy
- [x] If needed, I have included a [Changeset](https://github.com/changesets/changesets) and it follows [Semantic Versioning principles](https://semver.org/)
- [x] I have completed all of the above and have enabled `auto-merge` for this PR

### CODE REVIEW

**Review by lordpixel — CHANGES_REQUESTED** [review]
@rociobaena This needs to go to develop, instead of Main.

Also, the code is now frozen until  an upstream PR is merged into develop to close the release loop. Please allow me to run that process first.

**Comment by rociobaena** [comment]
> @rociobaena This needs to go to develop, instead of Main.
> 
> Also, the code is now frozen until an upstream PR is merged into develop to close the release loop. Please allow me to run that process first.

Manu, sorry. I didn't notice the branch was main. Thank you for pointing out it. Target changed.

**Comment by lordpixel** [comment]
@rociobaena this looks good, we only need to wait for the upstream PR to be merged.

File: packages/cascara/src/atoms/DatePicker/DatePicker.js
Comment (lordpixel) [inline packages/cascara/src/atoms/DatePicker/DatePicker.js:9]: @rociobaena let's all `picker` possible values in a `_globals.js` file:

```javascript
const PICKER_TYPE = {
    DATE: 'DATE',
    WEEK: 'WEEK',
    MONTH: 'MONTH',
    QUARTER: 'QUARTER',
    YEAR: 'YEAR'
};

// so in props we can do this:
picker: oneOf(Object.keys(PICKER_TYPE),
```

**Review by lordpixel — CHANGES_REQUESTED** [review]
@rociobaena this looks OK, we are just missing the unit tests and I left a comment.

**Review by odiseo42 — APPROVED** [review]

**Review by lordpixel — APPROVED** [review]
👍

**Pull request merged**
